### PR TITLE
Testing author label length limitation for Bitbucket server.

### PR DESCRIPTION
**Pull request by appleboy**
* Fixed some coding style using `gofmt` tool.
* Testing author label length limitation.

cc @josmo

Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

File: remote/bitbucketserver/convert.go [hunk 1]
```diff
@@ -67,15 +67,14 @@ func convertPushHook(hook *internal.PostHook, baseURL string) *model.Build {
...
 	commitType := refParts[1]
 
-
 	//Ensuring the author label is not longer then 40 for the label of the commit author (default size in the db)
 	authorLabel := hook.Changesets.Values[0].ToCommit.Author.Name
-	if (len(authorLabel) > 40) {
-		authorLabel = authorLabel[0:40]
+	if len(authorLabel) > 40 {
+		authorLabel = authorLabel[0:40] + "..."
 	}
```
Comment (josmo) [inline remote/bitbucketserver/convert.go]: If you want to do it that way I think it would have to be [0:37] + "..." :)

**Comment by josmo** [comment]
Rest looks good... :) I keep forgetting to fmt things :/  I guess I need to find a way to remember.

File: remote/bitbucketserver/convert.go (hunk 1 above)
Comment (appleboy) [inline remote/bitbucketserver/convert.go]: @josmo Done 👍

**Comment by josmo** [comment]
LGTM :)

**Comment by appleboy** [comment]
@josmo Try to install your editor plugin. Any editor can auto run `gofmt` when you save the changes.

**Comment by josmo** [comment]
Thanks @appleboy just set it up and I'm sure that will stop me from bugging others :)

**Pull request merged**